src/hotosm.yaml

```diff
@@ -0,0 +1,44 @@
+---
+description: >
+  HOT is an international team dedicated to humanitarian action and
+  community development through open mapping.
+  They provide map data to revolutionize disaster management, reduce risks,
+  and contribute to achieving SDGs.
+
+  HOT also provides training, equipment, knowledge exchange and field prjects.
+# At this point still unknown
+# See https://github.com/hotosm/hotosm-website/issues/626
```

Comment:
Maybe we should link to the organization page https://github.com/hotosm instead of one of their projects.
I can see a lot of BSD-2 and BSD-3 repos there.

On [their website](https://www.hotosm.org/community/), I read

> HOT is a global not-for-profit organization registered in the United States of America.
> 
> As part of our obligations as a 501(c)3 registered organization, we make our financial filings (known as “Form 990: Return of Organization Exempt From Income Tax”) available for public inspection. HOT’s 2012 - 2017 returns are posted below. Any further questions can be directed to the Board of Directors, specifically the Treasurer.

Not really related to OSS or not though. Weird that they don't have a document about that.

Reply:
The list mentioned in #37 kinda makes it seem as if all of them are OSS. HOTOSM seems to have a stricter policy on accepting PR than most and designated teams of volunteers.

However, they also have `hacktober` issues, which kinda means that they do accept PRs from third party people.